src/app.module.ts

```diff
@@ -21,6 +28,6 @@ import { join } from 'path';
     UserModule,
   ],
   controllers: [AppController],
-  providers: [AppService],
+  providers: [AppService, UserResolver],
 })
 export class AppModule {}
```

Comment:
@Nandita-Nikam no need to add UserResolver in providers  when you are adding UserModule in imports

Reply:
ok